platform/mellanox/mlnx-platform-api/sonic_platform/psu.py

```diff
@@ -304,7 +306,14 @@ def get_model(self):
         Returns:
             string: Model/part number of device
         """
-        return self.vpd_parser.get_model()
+        current_model = self.vpd_parser.get_model()
+        if current_model != 'N/A' and current_model != self.model and os.path.exists('/usr/share/sonic/platform/psu_sensors_conf_updater'):
```

Comment:
better run this code in container

Reply:
os.path.exists('/usr/share/sonic/platform/psu_sensors_conf_updater') would insure host calling will not go into this branch